syncano/exceptions.py

```diff
@@ -0,0 +1,31 @@
+class SyncanoException(Exception):
```

Comment:
My favorite way of creating a custom exceptions is
```
class SyncanoException(Exception):
    pass
```
Usually there is no need for reimplementing those built in methods.